package.json

```diff
@@ -40,6 +40,7 @@
   },
   "devDependencies": {
     "autoprefixer": "^6.3.6",
+    "babel-plugin-transform-object-assign": "^6.8.0",
```

Comment:
prefer to _limit_ our usage of babel plugins as much as possible. 
This particular does not _appear_ to have tests: https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-object-assign ... unless I'm missing something?